Mathlib/Topology/Algebra/OpenSubgroup.lean

```diff
@@ -12,8 +12,8 @@ import Mathlib.Topology.Sets.Opens
 /-!
 # Open subgroups of a topological group
 
-This files builds the lattice `OpenSubgroup G` of open subgroups in a topological group `G`,
-and its additive version `OpenAddSubgroup`.  This lattice has a top element, the subgroup of all
+This file builds the lattice `OpenSubgroup G` of open subgroups in a topological group `G` and its
```

Comment:
Please keep the comma. To my eyes, that actually changes meaning (and the new meaning is wrong).

Reply:
I think I see your point. Although, I think this sentence is ambiguous either way it is structured. I don't think the presence or the absence of the comma makes it clearer what "its" is referring to. So we might have to revisit this. But the dropped comma is reverted for now.